Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -1083,6 +1089,46 @@ void HackStudioWidget::initialize_menubar(GUI::Menubar& menubar)
     create_help_menubar(menubar);
 }
 
+void HackStudioWidget::update_statusbar()
+{
+    StringBuilder builder;
+    builder.appendff("Line: {}, Column: {}", current_editor().cursor().line() + 1, current_editor().cursor().column());
+
+    if (current_editor().has_selection()) {
+        size_t first_line;
+        size_t last_line;
+        auto selection = current_editor().normalized_selection();
+        if (!selection.is_valid()) {
+            first_line = current_editor().cursor().line();
+            last_line = current_editor().cursor().line();
+        } else {
+            first_line = selection.start().line();
+            last_line = selection.end().line();
+        }
+        if (first_line == last_line || selection.start().column() == 0 || selection.start().column() != 0)
+            last_line += 1;
+
+        builder.appendff("        Selected: {} {}", last_line - first_line, (last_line - first_line) != 1 ? "Lines" : "Line");
```

Comment:
Please use multiple statusbar segments instead of adding a bunch of whitespace to the first segment :)